tests/behaviour/expectations/semantic_whitelist.ts

```diff
@@ -1607,7 +1607,7 @@ const tests: string[] = [
     // 'tests/behaviour/solidity/test/libsolidity/semanticTests/constructor_with_params_inheritance_2.sol',
     // 'tests/behaviour/solidity/test/libsolidity/semanticTests/empty_contract.sol', // test suite cannot call non-existant function
     // 'tests/behaviour/solidity/test/libsolidity/semanticTests/constructor_inheritance_init_order_3_legacy.sol', // We don't support the legacy bug behaviour and instead implement viaYul behaviour.
-    // 'tests/behaviour/solidity/test/libsolidity/semanticTests/interface_inheritance_conversions.sol', // REQUIRES CAIRO 0.9 uses new
+    // 'tests/behaviour/solidity/test/libsolidity/semanticTests/interface_inheritance_conversions.sol', // Address clash when deploying the same contracts
```

Comment:
I'm a little uncomfortable with how many failing tests there are here for a PR up for merging (though this may also be because I find it hard to tell from these error messages which are expected failures (WILL NOT SUPPORT) and which are unexpected). Ultimately though, I'll leave it up to you and @GuyMcComb as to how many if any need fixing before this PR is merged

Reply:
Looks like all the tests that fail because of 'value option' should be changed to will not support? And I assume we have created bug tickets for "Fails because it WARP allows a contract to call itself during creation (And it shoudl not)"

Reply:
Modified those to `WILL NOT SUPPORT due to reason` to be more clear